reports/jshint-lines-of-shame.js

```diff
@@ -0,0 +1,48 @@
+// TODO: We need to figure out whether React will be supported natively for a reporter or if they
+// will be required to pre-compile
```

Comment:
In order for a reporter to use `*.jsx` files, the author must either pre-compile them to regular javascript before publishing to npm OR he/she would rely on krabby having a tool like babel installed that allows `.jsx` compilation to happen at run time.